environments/dotnet20/builder/nugetSettings.json

```diff
@@ -0,0 +1,39 @@
+{
+  "NugetFolder": "Nugetdownload",
+  "DisableCache": false,
+  "CSVDirectory": "logs",
+  //"NugetPackageRegEx": "\\:?\\s*(?<package>[^:\\n]*)(?:\\:)?(?<version>.*)?\\n",
```

Comment:
Just notice this file and there are left comments. I'm checking why synk check failed, at the same time please remove unnecessary comments.

Reply:
Nuget based  regex is being used as per regex in  builderSettings.json and not in nugetSettings.json (that nugetSettings.json regex is obsolete ) , removed it.
However , there is some other comments in this file which are kind of guidance for users on how to use private feed , i am keeping that comment for now so that user may understand it and add their private feeds for nuget.

Reply:
ok, I originally thought it was sensitive info that pushed accidentally.